docs/HowToGuides.md

```diff
@@ -1,5 +1,63 @@
 # How To ...
 
+### Create a multi-step PR
+
+Big PRs (i.e. PRs touching more than 100 lines) are hard to review. 
+Using the _multi-step PR_ format explained below allows the review to be done incrementally.
+
+1. Dev creates the PR branch consisting of a series of commits corresponding to a sequence of logical steps, 
+   each step advancing the code base a step towards the end goal of the PR. The build should pass at each step.
+   <details>
+   <summary>Instructions</summary>
+   
+   * There Ideal approach for creating a multi-step commit sequence: 
+     * Plan the PR as a sequence of logical steps.
+     * Implement each step, committing each step as a separate commit. 
+       * Write detailed commit messages as if you were explaining your commit to a future developer.
+       
+        > Writing detailed commit messages is especially important for multi-step PRs because the commits will be 
+        > preserved after merging (i.e. the branch will be merged using a merge commit instead of 'squash and merge')
+        
+       * You may have to `squash` trivial commits or reorder commits to make the steps more logical. 
+   * Alternative approach:
+     * Implement the fix, committing after each significant change.
+     * After the fix is complete, refactor commits so the you have a commit sequence similar to what you would have 
+       achieved in the ideal approach given above.
+   </details>
+   
+1. When the PR is ready for reivew, dev posts a summary of commits using the CanIHasReview tool.
+   (created by [@pyokagan](https://github.com/pyokagan/))  
```

Comment:
As a reader, it's kind of annoying to see this actually :-P

Maybe just link `CanIHasReview` to https://github.com/pyokagan/canihasreview/ ? Enterprising readers should be able to figure out the rest.